mso/resource_mso_schema_site_anp_epg.go

```diff
@@ -75,8 +89,8 @@ func resourceMSOSchemaSiteAnpEpgImport(d *schema.ResourceData, m interface{}) ([
 
 	stateSite := get_attribute[2]
 	found := false
-	stateAnp := get_attribute[4]
-	stateEpg := get_attribute[6]
+	stateAnp := get_attribute[6]
+	stateEpg := get_attribute[8]
```

Comment:
Why did this change?

Reply:
This was actually a bug. It was supposed to be this way.